packages/plugins/chainflip/src/broker.ts

```diff
@@ -230,8 +229,9 @@ const fundStateChainAccount =
     evmToolbox: ReturnType<typeof ETHToolbox>;
     stateChainAccount: string;
     assetValue: AssetValue;
+    fromAddress: string;
   }) => {
-    if (assetValue.symbol !== "FLIP") {
+    if (assetValue.symbol !== "ETH.FLIP-0x826180541412D574cf1336d22c0C0a287822678A") {
```

Comment:
```suggestion
    if (assetValue.toString() !== "ETH.FLIP-0x826180541412D574cf1336d22c0C0a287822678A") {
```

Reply:
symbol will always fail, no?

Reply:
I would probably just create the assetValue for the asset we are testing here and then compare the to String value to be sure - doing this due to change to case sensitive addresses this week

Reply:
good call, I ended up just using asset value and eq method